db/seeds.rb

```diff
@@ -1,8 +1,14 @@
 unless User.find_by(email: 'adam@codingzeal.com')
 	User.create(
-		email: 'adam@codingzeal.com',
+		[{email: 'adam@codingzeal.com',
```

Comment:
Indentation and is there a reason we're adding another user?

Reply:
deleted another user